### ISSUE TITLE
Is Dragdealer have destroy method or analog to stop calling requestAnimationFrame?

### ISSUE THREAD

**Issue by Dok11** [skidding/dragdealer#133]
It's need for SPA website least.
And for example user can close modal window with Dragdealer element, but requestAnimationFrame still calling.

![image](https://user-images.githubusercontent.com/2697890/52223904-c1f9a380-28b7-11e9-9aa2-0776ed3791f4.png)

**Comment by TonyBogdanov** [comment]
+1

**Comment by TonyBogdanov** [comment]
The only thing I've found is `unbindEventListeners()`, which seems to release all listeners and stop the `requestAnimationFrame` loop, but it won't reset the HTML structure. Good enough for me, though.

**Comment by ovidiuch** [comment]
Indeed, `unbindEventListeners` should destroy a Dragdealer instance.
